test/EigenLayerIntegration.t.sol

```diff
@@ -100,41 +92,35 @@ contract EigenLayerIntegraitonTest is TestSetup, ProofParsing {
         createdForks[forkURL] = vm.createFork(forkURL);
         vm.selectFork(createdForks[forkURL]);
 
-        _upgrade_etherfi_node_contract();   
-        _upgrade_etherfi_nodes_manager_contract(); 
-        _upgrade_staking_manager_contract();
-        _upgrade_liquidity_pool_contract();
-        _upgrade_auction_manager_contract();
-        _upgrade_node_oeprator_manager_contract();
-        _upgrade_etherfi_oracle_contract();
-
         setupRoleRegistry();
     }
+    
+    // Broken by PEPE upgrades
+    // TODO: Update test post PEPE upgrade
+    // function test_fullWithdraw_812() public {
```

Comment:
is there any way to keep any unit test on `fullWithdraw`?

if you are going to add a new one, we can remove it? @solipsis

Reply:
just fyi the `fullWithdraw` function call in this test reverts with the internal  `_distributePayouts` call attempting to distribute more ETH than there is in the Node contract